R/hoist.R

```diff
@@ -133,7 +133,7 @@ hoist <- function(.data,
   out
 }
 
-check_pluckers <- function(...) {
+check_pluckers <- function(..., .call = caller_env()) {
```

Comment:
```suggestion
check_pluckers <- function(..., .error_call = caller_env()) {
```

Reply:
According to your conventions this should be `.call`? Since this is a check function?

Reply:
Yea my bad